src/amber/router/params_parser.cr

```diff
@@ -115,10 +115,18 @@ module Amber::Router
     def parse_multipart
       HTTP::FormData.parse(request) do |upload|
         next unless upload
+        # Note:
```

Comment:
I don't think we need this note here. In the case that a file is submitted via a multipart form filename is there. If not it isn't. This check is important to make sure that empty files don't end up in the Files hash. It also saves us from time on parsing.

Reply:
The reason I added the note is because of this line: `The filename is always optional and must not be used blindly by the application:`